shared/js/media/crop_resize_rotate.js

```diff
@@ -255,10 +259,16 @@ function cropResizeRotate(blob, cropRegion, outputSize, outputType,
     }
 
     // Make sure the outputType is valid, if one was specified
-    if (outputType && outputType !== JPEG && outputType !== PNG) {
+    if (outputType && outputType !== JPEG &&
+        outputType !== PNG && outputType !== EXIF) {
       callback('unsupported outputType: ' + outputType);
       return;
     }
+    var preserveExif = false;
+    if (outputType === EXIF) {
+      preserveExif = true;
```

Comment:
Actually, it occurs to me now that preserveExif should only be true if `blob.type === JPEG` Otherwise there is no exif to preserve.  We probably don't have any code that calls cropResizeRotate with outputType EXIF if the input blob is not a JPEG, but to be completely correct, we should check for that here so we don't try to read metadata from a PNG later on.